libraries/lib-basic-ui/BasicUI.h

```diff
@@ -311,10 +312,10 @@ inline std::unique_ptr<ProgressDialog> MakeProgress(
  */
 inline std::unique_ptr<GenericProgressDialog> MakeGenericProgress(
    const WindowPlacement &placement,
-   const TranslatableString &title, const TranslatableString &message)
+   const TranslatableString &title, const TranslatableString &message, int style = 0x0002 | 0x0008 | 0x0020)
```

Comment:
One last thing: `lib-basic-ui` is a toolkit-neutral library, better define `enum` and map it to a framework-specific value in the library that implements `BasicUI` interface, which is `lib-wx-init`